site/_posts/2020-09-23-jerry-hopper-fusionauth.md

```diff
@@ -1,7 +1,7 @@
 ---
 layout: blog-post
-title: Contaim adopts FusionAuth to manage their auth lifecycle
-description: Contaim CEO Jerry Plunkett discusses how FusionAuth has helped his business.
+title: Jerry loved FusionAuth so much he built an Arm image for it
```

Comment:
Arm -> ARM ?

I could only find `Arm` when referring to `Arm Holdings` but as an arch I think it is always `ARM`?

Reply:
nice catch!